### PR TITLE
 Flytectl get started added for running example

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
- Added Get started guide for registering first workflow 

For testing examples are available at https://github.com/flyteorg/flytesnacks/releases/tag/v0.2.89 

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
https://github.com/flyteorg/flytesnacks/pull/166
https://github.com/flyteorg/flyte/pull/942

### CODE REVIEW

**Review by pmahindrakar-oss — APPROVED** [review]

**Comment by pmahindrakar-oss** [comment]
@evalsocket You would need to sign the commits using the GPG keys for this repo. Hence the merge is blocked.
Whenver you get time you can setup this for your account . Follow the instructions from this page https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification
which is the same one from merging is blocked is label above.

Also if possible once you are able to do this , Can you add this to Getting started page so the contributors are aware of this new change.

**Comment by evalsocket** [comment]
@pmahindrakar-oss let's merge it after flytesnack release

**Review by pmahindrakar-oss — DISMISSED** [review]

**Comment by pmahindrakar-oss** [comment]
Ping me or @kumare3 to merge it once the aritifacts are available for flytesnacks. And also instead of going of a static version can this pick whats the latest available.

**Comment by evalsocket** [comment]
@pmahindrakar-oss @kumare3 flytesnacks examples are available in new release, We can merge it

**Review by katrogan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,7 +24,20 @@ Generating docs locally can be accomplished by running make gendocs from within
...
+### Create a sandbox cluster 
+```bash
+$ docker run --rm --privileged -p 30081:30081 -p 30082:30082 -p 30084:30084 ghcr.io/flyteorg/flyte-sandbox
+```
+
+### Register your first workflow
+
+```bash
+# Run Core workflows
+$ flytectl register files  https://github.com/flyteorg/flytesnacks/release/releases/download/v0.2.89/flytesnacks-core.tgz -d development  -p flytesnacks --archive
```
Comment (katrogan) [inline README.md:38]: can we have this download the latest release version?

Comment (kumare3) [reply README.md:38]: @katrogan that would be hard, as the semantics of the release URL are not know to `flytectl`
IMO @pmahindrakar-oss, @katrogan we should probably have a different cli target
```
flytectl register examples --version v0.2.89
# OR defaults to latest if --version is not provided?
```

**Comment by kumare3** [comment]
this did not work for me 
```
flytectl register files https://github.com/flyteorg/flytesnacks/release/releases/download/v0.2.89/flytesnacks-core.tgz -d development -p flytesnacks --archive
INFO[0000] Using config file: [/Users/ketanumare/.flyte/config.yaml]
INFO[0000] Config section [root] updated. No update handler registered.  src="viper.go:349"
INFO[0000] Config section [logger] updated. Firing updated event.  src="viper.go:351"
Error: unexpected EOF
```

```
➜ flytectl version
INFO[0000] Using config file: [/Users/ketanumare/.flyte/config.yaml]
INFO[0000] Config section [admin] updated. Firing updated event.  src="viper.go:351"
INFO[0000] Config section [storage] updated. No update handler registered.  src="viper.go:349"
INFO[0000] Config section [adminutils] updated. No update handler registered.  src="viper.go:349"
INFO[0000] Config section [root] updated. No update handler registered.  src="viper.go:349"
INFO[0000] Config section [logger] updated. Firing updated event.  src="viper.go:351"
{
  "App": "flytectl",
  "Build": "cdc1938",
  "Version": "0.1.8",
  "BuildTime": "2021-05-26 15:26:36.030606 -0700 PDT m=+0.013616840"
}{
  "App": "controlPlane",
  "Build": "db32bb3",
  "Version": "v0.5.4",
  "BuildTime": "2021-05-21 00:11:58.498302524 +0000 UTC m=+0.028839574"
}
```

**Comment by kumare3** [comment]
The reason for the failure - bad path @evalsocket - 
```
flytectl register files -d development -p flytesnacks --archive https://github.com/flyteorg/flytesnacks/releases/download/v0.2.89/flytesnacks-core.tgz
```
worked!

**Review by pmahindrakar-oss — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pmahindrakar-oss) [reply README.md]: Also now that we have multiple modules and each with its own archive then we would additionally need option for that too. eg : https://github.com/flyteorg/flytesnacks/releases/download/v0.2.89/flytesnacks-integrations-hive.tgz

**Pull request merged**